WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -26,6 +25,7 @@ private static SQLiteDatabase getWritableDb() {
     public static void createTables(SQLiteDatabase db) {
         db.execSQL("CREATE TABLE " + PEOPLE_TABLE + " ("
                 + "person_id               INTEGER DEFAULT 0,"
+                + "site_id                 TEXT,"
```

Comment:
I can be wrong here but, my impression is that the rest of the (local) tables use some form of `blog_id` instead of `site_id` even though the remote API documentation uses `site id`. May I suggest renaming this to `blog_id` to match the rest of the app? Cheers!
This will make it more clear against the `local_blog_id` as well.